icons/flame.svg

```diff
@@ -0,0 +1,13 @@
+<svg
+  xmlns="http://www.w3.org/2000/svg"
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+    <path d="M8.49997 14.5C9.88068 14.5 11 13.3807 11 12C11 10.6193 10.5 10 9.99997 9C8.9283 6.85666 9.77608 4.9459 12 3C12.5 5.5 14 7.9 16 9.5C18 11.1 19 13 19 15C19 18.866 15.866 22 12 22C8.13401 22 5 18.866 5 15C5 13.847 5.43297 12.706 5.99997 12C5.99997 13.3807 7.11926 14.5 8.49997 14.5Z" />
```

Comment:
Minor nitpick here @ericfennis 😅 Indentation of 2 not 4

```suggestion
  <path d="M8.49997 14.5C9.88068 14.5 11 13.3807 11 12C11 10.6193 10.5 10 9.99997 9C8.9283 6.85666 9.77608 4.9459 12 3C12.5 5.5 14 7.9 16 9.5C18 11.1 19 13 19 15C19 18.866 15.866 22 12 22C8.13401 22 5 18.866 5 15C5 13.847 5.43297 12.706 5.99997 12C5.99997 13.3807 7.11926 14.5 8.49997 14.5Z" />
```

Reply:
Sorry my fault, I know it's not the first time 😅 . I think we need an `.editorconfig`. 
My tabs are constantly switching spaces.